doc/source/basics.rst

```diff
@@ -302,23 +302,17 @@ To evaluate single-element pandas objects in a boolean context, use the method
 
 .. warning::
 
-   You might be tempted to do the following:
+   Using a DataFrame as a condition will raise errors,
```

Comment:
What was the point of changing this? We always ask changes to have one focus to keep diff minimal. Would prefer if you could back out anything unrelated to the flake8 and do in a separate PR